src/global_stats.c

```diff
@@ -53,6 +54,10 @@ void FieldsGlobalStats_UpdateStats(FieldSpec *fs, int toAdd) {
   }
 }
 
+void FieldsGlobalStats_UpdateIndexError(FieldType field_type, int toAdd) {
+  FieldIndexErrorCounter[INDEXTYPE_TO_POS(field_type)] += toAdd;
```

Comment:
adding int to size_t

Reply:
we are doing it to allow negative `toAdd `